spec/sections/standard.adoc

```diff
@@ -196,28 +197,28 @@ The following annotated snippet provides quick reference to the use of Interface
 [source,javascript]
 ----
 {
-    "cmd": "/app/job.sh ${INPUT_FILE} ${OUTPUT_DIR}", // <1>
-    "inputData": { "files": [ { "name": "INPUT_FILE", ... }, ... ] }, // <2>
-    "outputData": { ... }, // <3>
+    "command": "/app/job.sh ${INPUT_FILE} ${OUTPUT_DIR}", // <1>
+    "inputs": { "files": [ { "name": "INPUT_FILE", ... }, ... ] }, // <2>
+    "outputs": { ... }, // <3>
     "mounts": [ ... ], // <4>
     "settings": [ ... ] // <5>
 }
 ----
 <1> Optional string indicating the job arguments. Reference <<environment-variables>> for clarification on what the
 implementing framework MUST provide. Linux shell escaping MAY be needed in the case of special characters.
-<2> Optional <<inputdata-section>>. This is the means to inject external data into the job container.
-<3> Optional <<outputdata-section>>. This is the means to capture results from the job container.
+<2> Optional <<inputs-section>>. This is the means to inject external data into the job container.
+<3> Optional <<outputs-section>>. This is the means to capture results from the job container.
 <4> Optional <<mounts-section>>. This defines any directories that need to be mounted into the job container.
 <5> Optional <<settings-section>>. This defines any environment specific settings needed at run time.
 
-[[inputdata-section, InputData Object]]
-====== InputData Object
-The InputData object is the member responsible for indicating all mutable content available to the Seed image at
+[[inputs-section, Inputs Object]]
+====== Inputs Object
+The Inputs object is the member responsible for indicating all immutable content available to the Seed image at
```

Comment:
I wonder if this should be reworded to talk about input data. Mounts are technically another way to provide "immutable content available to the Seed image at runtime".